packages/components/src/List/ListComposition/Manager/hooks/useLocalStorage.hook.js

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react';
+
+export default function useLocalStorage(localStorageKey, initialValue) {
+	const [value, setValue] = useState(localStorage[localStorageKey] || initialValue);
+	const onChange = newValue => {
+		setValue(newValue);
```

Comment:
the intermediate in memory object maybe useful to check that newValue !== oldValue and be sure to  prevent writting it if not needed

Reply:
I changed the implementation to use react-use instead of implementing it, as @frassinier suggested